src/content/en/2021/resource-hints.md

```diff
@@ -1,20 +1,609 @@
+
+#### `fetch`
+
+Preload may be used to initiate a request to retrieve data which you know is critical to the rendering of the page, such as a JSON response or stream.
+
+#### `image`
+
+Preloading images may help improve the LCP score when the image is not included in the initial HTML, such as a CSS `background-image`.
+
+### The `crossorigin` attribute
+
+The `crossorigin` attribute is used to indicate whether [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) must be used when fetching the requested resource. This could apply to any resource type, but it is most commonly associated with font files as they should always be requested using CORS.
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>value</th>
+        <th>desktop</th>
+        <th>mobile</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>not set</td>
+        <td class="numeric">66.6%</td>
+        <td class="numeric">65.9%</td>
+      </tr>
+      <tr>
+        <td><code>crossorigin</code> (or equivalent)</td>
+        <td class="numeric">14.5%</td>
+        <td class="numeric">13.5%</td>
+      </tr>
+      <tr>
+        <td><code>use-credentials</code></td>
+        <td class="numeric">&lt; 0.1%</td>
+        <td class="numeric">&lt; 0.1%</td>
+      </tr>
+    </tbody>
+  </table>
+
+<figcaption>{{ figure_link(caption='<code>rel="preload" crossorigin</code> attribute values.', sheets_gid="1246058294", sql_file="hint_attribute_usage.sql") }}</figcaption>
+</figure>
+
+#### `anonymous`
+
+The default value when no value is specified, `anonymous` will set the credentials flag to <a hreflang="en" href="https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy">`same-origin`</a> and is required when downloading resources protected by CORS. It is also a <a hreflang="en" href="https://drafts.csswg.org/css-fonts/#font-fetching-requirements">requirement</a> when downloading font files—even if they are on the same origin! If you omit the `crossorigin` attribute when the eventual request for the preloaded resource uses CORS, you will end up with a duplicate request since it won't match in the preload cache.
```

Comment:
```suggestion
The default value when no value is specified, `anonymous`, will set the credentials flag to <a hreflang="en" href="https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy">`same-origin`</a> and is required when downloading resources protected by CORS. It is also a <a hreflang="en" href="https://drafts.csswg.org/css-fonts/#font-fetching-requirements">requirement</a> when downloading font files—even if they are on the same origin! If you omit the `crossorigin` attribute when the eventual request for the preloaded resource uses CORS, you will end up with a duplicate request since it won't match in the preload cache.
```

Reply:
I think there shouldn't be a comma here.

Reply:
I think there should be.

But think it would be better worded as:

```suggestion
The default value when no value is specified is `anonymous` and this value will set the credentials flag to <a hreflang="en" href="https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy">`same-origin`</a>. It is required when downloading resources protected by CORS. It is also a <a hreflang="en" href="https://drafts.csswg.org/css-fonts/#font-fetching-requirements">requirement</a> when downloading font files—even if they are on the same origin! If you omit the `crossorigin` attribute when the eventual request for the preloaded resource uses CORS, you will end up with a duplicate request since it won't match in the preload cache.
```